### ISSUE TITLE
Window menu bugged

### ISSUE THREAD

**Issue by tweety231323** [vpinball/vpinball#840]
**Describe the bug**
Open / close tables. Window menu shows the tables as still open.

**To Reproduce**
1. File - New - New table
2. File - Close
3. Window -> shows "1 table1".

This gets worse when you open several tables and close Table1, Table2 etc. The Window menu stops working.

**Versions & Settings**
 - VPX: VPinballX_GL-10.8.0-1508-6f84b57-Debug-win-x64
 - Windows version 11 Home

Example:
Table2 now shows Table3, Table3 shows Table4 and Table4 also shows Table4.
![2023-09-29 14_41_45-Visual Pinball -  Table3](https://github.com/vpinball/vpinball/assets/98708681/5ebf87b3-08f1-416f-b6db-d80f53d21b57)

**Comment by tweety231323** [comment]
Another example:
loaded 9 tables, tried to load #10, got the message "maximum number of tables loaded".
Closed 9 tables, Window menu shows 9 open tables.
![2023-09-30 18_10_39-Visual Pinball](https://github.com/vpinball/vpinball/assets/98708681/70bcf0c9-cd51-4a8a-b3df-111f9e3a803a)

**Comment by toxieainc** [comment]
Yes, this is a very old bug that appeared when switching to Win32++. We don't know yet how to solve that, but thanks for tracking.
